Data_size_details.Rmd

```diff
@@ -0,0 +1,64 @@
+
+
+## Genomics Data {-}
+
+Matt Fitzgibbon and Andy Marty from Genomics Shared Resources have put together a table of file sizes generated by common genomics assays done at Fred Hutch. These estimates are only approximate, as the actual file sizes can vary considerably. Per-samples sizes are averaged from samples of at least three representative runs of the given type (except for 10x Multiome where two runs were checked).
+
+Assay | File Type | Per-sample Size | Per-run Size | Public Repository | Private Repository | Notes |
+|------------------|------------------|------------------|------------------|------------------|------------------|------------------|
+Bulk RNA-seq | Paired Fastq | 2-4G | highly variable | GEO/SRA | dbGaP/SRA | Depends on library prep & goals |
+RNA Exome | Paired Fastq | 3G | highly variable | GEO/SRA | dbGaP/SRA | |
+Whole Exome | Paired Fastq | 3G | highly variable | GEO/SRA | dbGaP/SRA | HS platform dependent |
+CRISPR | Single Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | sgRNA library dependent |
+CUT&RUN | Paired Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
+CUT&Tag | Paired Fastq | ≥500M | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
+ChIP-seq | Fastq | 0.5-5G | highly variable | GEO/SRA | dbGaP/SRA | Ab dependent |
+ATAC-seq | Fastq | 3-5G | highly variable | GEO/SRA | dbGaP/SRA | |
+10x scRNA-seq | Paired Fastq | 10G | highly variable | GEO/SRA | dbGaP/SRA | Target cell number dependent |
+10x Multiome | Paired Fastq | ≥20G | highly variable | GEO/SRA | dbGaP/SRA | Target nuclei number dependent |
+10x Visium | Paired Fastq | ≥5G | highly variable | GEO/SRA | dbGaP/SRA | Spots under tissue dependent |
+Small Genome | Paired Fastq | ≥2G | highly variable | GEO/SRA | N/A | Genome size dependent |
+PacBio Amplicon | CCS BAM | 0.5-20G | highly variable | GEO/SRA | N/A | Amplicon size & target depth dependent |
+PacBio Small Genome | CCS BAM | highly variable | highly variable | GEO/SRA | N/A | Genome size dependent |
+
+
+## Imaging Data {-}
+
+File sizes for medical imaging data vary greatly depending on both the technology used and the organ being imaged. These are some general estimates you can use as a guideline when considering your data management and storage needs. These tables are borrowed from the [ITN Computing for Cancer Informatics Course](https://jhudatascience.org/Computing_for_Cancer_Informatics/index.html).
+
+
+Here is an table of average file sizes for various medical imaging modalities from @liu_imaging_2017:
```

Comment:
did we add this reference to the book.bib too?

Reply:
No, I am glad you caught that!